docs/scos/dev/tutorials-and-howtos/howtos/howto-delete-store-with-ds.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "HowTo: Delete store when using the dynamic store feature"
+description: This document shows how to delete store when the system is running with a dynamic store feature.
+past_updated: Jun 26, 2023
+template: howto-guide-template
+---
+
+This document describes the steps to consider how manually delete store in spryker when using dynamic store.
+
+## When it might be useful
+
+Often, the store that was created earlier becomes unnecessary. It can be a store that has been used for a brand, product, holiday, or marketing promotions or events аnd ending experiments with no production environment. 
+
+On the technical side, removing such a store helps:
+1. Increasing the speed of the store and deleting redundant data.
```

Comment:
Editor's note: how does deleting a store help with increasing its speed?

Reply:
Data of stores take up a lot of space on the server. Deleting a store frees up space on the server reducing the load on the server for indexing and search in database. Same for search service (Elasticsearch) and storage service (Redis).